### PR TITLE
Fix about pages

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes About part of #2219 

Changes: 
- [x] Fix mobile view of all about pages
- [x] Integrate styled components and get rid of CSS.
- [ ] Use rem instead of px. {Will take it up in next PR}

Demo Link: https://pr-2264-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@fragm3 Made the changes. Please review.

**Comment by harsh-jindal** [comment]
@fragm3 Please check now.
Screenshot 2019-06-16 at 11 42 56 AM
Screenshot 2019-06-16 at 11 43 20 AM

**Comment by fragm3** [comment]
@harsh-jindal still present

**Comment by harsh-jindal** [comment]
@fragm3 Fixed

**Comment by harsh-jindal** [comment]
@fragm3 The overview page was always like this. I have kept it the same. I thought it was intentionally kept like that. 
@akshatnitd Can you help here ? Shall I center align it or leave it as it is ?

**Comment by akshatnitd** [comment]
It should be center aligned

**Comment by harsh-jindal** [comment]
@akshatnitd @fragm3 I have made the changes.

**Comment by harsh-jindal** [comment]
@fragm3 Done

File: src/components/Overview/Overview.react.js [hunk 1]
```diff
@@ -30,21 +30,504 @@ import Action from '@material-ui/icons/ChatBubble';
...
+  width: 50%;
+  @media (max-width: 1000px) {
+    width: 100%;
+  }
+`;
+
+const CustomDescription = styled.div`
+  ${commonDesc};
+  width: 100%;
+  @media (max-width: 1000px) {
```
Comment (fragm3) [inline src/components/Overview/Overview.react.js]: Remove

File: src/components/Blog/Blog.react.js [hunk 2]
```diff
@@ -109,6 +95,86 @@ const BlogFooter = styled.div`
...
+  width: 100%;
+`;
+
+const BlogFooterLink = styled.a`
+  white-space: nowrap;
+`;
+
+const OverlayLink = styled.a`
+  &&& {
+    ${LinkStyle};
```
Comment (fragm3) [inline src/components/Blog/Blog.react.js]: Why?

Comment (harsh-jindal) [reply src/components/Blog/Blog.react.js]: @fragm3 This is done to override the style from `.section-blog a` CSS.

**Comment by harsh-jindal** [comment]
@fragm3 Made the changes

**Comment by harsh-jindal** [comment]
@fragm3 Made the changes

**Comment by harsh-jindal** [comment]
@fragm3 @samagragupta Fixed.

**Pull request merged**